website/content/partials/telemetry-metrics/vault/replication/rpc/standby/server/wrap_token_request.mdx

```diff
@@ -0,0 +1,5 @@
+### vault.replication.rpc.standby.server.wrap_token_request ((#vault-replication-rpc-standby-server-wrap_token_request))
+
+Metric type | Value | Description
+----------- | ----- | -----------
+summary     | ms    | Time required to complete a wrap token request on the standby node
```

Comment:
```suggestion
summary     | ms    | Time required to service a wrap token request from a standby node
```

Can you apply this same pattern to the other rpc.standby.server metrics?

Reply:
fixed on all the standby.server metrics